vcstool/clients/git.py

```diff
@@ -413,6 +426,10 @@ def import_(self, command):
             cmd_checkout = [
                 GitClient._executable, 'checkout', checkout_version, '--']
             result_checkout = self._run_command(cmd_checkout)
+            if result_checkout['returncode']:
+                if self.get_git_version() < (1, 8, 4, 3):
```

Comment:
Throws:
`Invocation of command 'import' on client 'git' failed: TypeError: '<' not supported between instances of 'list' and 'tuple' (/usr/local/lib/python3.6/site-packages/vcstool/clients/git.py:430)`

```suggestion
                if self.get_git_version() < [1, 8, 4, 3]:
```

Reply:
Thanks for the fix. Did this make it work for you with an older version of git?

Reply:
Works great, thank you for the prompt response.

Reply:
Thanks for the feedback and testing.